fares-data-build-tool/create-data
# feat: cfd-428 logic for the needs attention alert

# Description

-   Each service that has products where one or more of the products have a matching service id in the `servicesRequiringAttention` field in the `products` table, show a 'NEEDS ATTENTION' tag.

# Testing instructions

-   Manually add a service Id to the `servicesRequiringAttention` field on the `products` table and check to see if the tag appears for that service.

# Type of change
